Makefile

```diff
@@ -42,9 +48,6 @@ composer:
 	@rm -f ./composer-setup.php
 
 protogen:
-ifeq (,$(findstring php-5.6.,$(PHP_VERSION)))
-	$(error protogen target should be run with PHP version 5.6.X instead of $(PHP_VERSION))
-endif
```

Comment:
@lukebakken I was able to run the protogen tool with PHP 7.0.2 without issue on my Mac.